pkgs/development/compilers/llvm/rocm/default.nix

```diff
@@ -29,6 +29,7 @@ let
   # Stage 1
   # Base
   llvm = callPackage ./llvm.nix {
+    requiredSystemFeatures = [ "big-parallel" ];
```

Comment:
I'm guessing this is only recognized by Hydra? What exact system resources does `big-parallel` correspond to?

Reply:
I'm not actually 100% sure what it corresponds to. I feel like it must be documented somewhere, but I can't find anything. It's up to the builder decide whether or not it supports `big-parallel` and it's intended as a tag dedicated for building large projects like llvm, firefox, libreoffice, ...

Reply:
This is a Nix remote builder feature. You can check in nixos-org configuration what big-parallel means.

Reply:
Oh thanks!! So then these are the specs for a normal build: https://deploy.equinix.com/product/servers/c3-medium, and these are the specs for a big-parallel build: https://deploy.equinix.com/product/servers/m3-large.